CGPs/0018.md

```diff
@@ -0,0 +1,35 @@
+# CGP [0018]: Update baselineQuorumFactor and Referendum Stage Duration
```

Comment:
You probably want to also split the titles

Reply:
Great catch 😂